lib/indico.js

```diff
@@ -112,6 +112,10 @@ var api_request = function (api, batch) {
     }
     var batch = batch || Object.prototype.toString.call(data).indexOf("Array") > -1;
 
+    if (api.type === "training" && !settings.resolvePrivateCloud(config, indico.cloud)) {
```

Comment:
Can private cloud not be set as a method parameter? If so, this won't catch that case.